Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1

```diff
@@ -1,4 +1,135 @@
-﻿function Get-TestAgentType([string] $Version)
+﻿function isAutoLogonDisabled()
+{
+    #check 64 bit or 32 bit
+	if ([IntPtr]::Size -eq 8)
+	{
```

Comment:
done

---
In reply to: [103681498](https://github.com/Microsoft/vsts-tasks/pull/3701#discussion_r103681498) [](ancestors = 103681498)